NEWS

```diff
@@ -8,6 +8,7 @@ Enhancements
  - Appointment color and importance work now between Outlooks
 
 Bug fixes
+ - Sent mails are not longer automatically copied to Drafts folder using Outlook
```

Comment:
imho we are doing it wrong this `news` file, I'd just stop updating because this is not useful for end users (I understand this message, because I know the bug, an end user won't understand this at all)